lib/logstash/outputs/base.rb

```diff
@@ -103,14 +103,14 @@ def output?(event)
       return false if !event["tags"]
       @include_method = :any?
       if !@tags.send(@include_method) {|tag| event["tags"].include?(tag)}
-        @logger.debug? and @logger.debug("outputs/#{self.class.name}: Dropping event because tags don't match #{@tags.inspect}", event)
+        @logger.debug? and @logger.debug(["outputs/#{self.class.name}: Dropping event because tags don't match #{@tags.inspect}", event])
         return false
       end
     end
 
     if !@exclude_tags.empty? && event["tags"]
       if @exclude_tags.send(@exclude_method) {|tag| event["tags"].include?(tag)}
-        @logger.debug? and @logger.debug("outputs/#{self.class.name}: Dropping event because tags contains excluded tags: #{exclude_tags.inspect}", event)
+        @logger.debug? and @logger.debug(["outputs/#{self.class.name}: Dropping event because tags contains excluded tags: #{exclude_tags.inspect}", event])
         return false
```

Comment:
We fixed fixed another bug this week on that particular line.. we should have tests for this regression. :boom:
See commit: https://github.com/elasticsearch/logstash/commit/19bfe8c14acbd0453724219a70f697c7578610e8

@wiibaa could you rebase this with the current master and It would be nice to migrate this to the new `logger` hash syntax. Similar to this:

```ruby
@logger.debug? and @logger.debug("Ouput dropping event because tags contains excluded tags",  :exclude_tags => @exclude_tags, :output => self.class.name, :event => event)
```